### DISCUSSION TITLE
preproc_batch_tops

### DISCUSSION THREAD

**Discussion by AliRoozban** [gmtsar/gmtsar#313] [Q&A]
Hi,
After selecting the master image and moving it to the first line of the data.in, should step "preproc_batch_tops.csh data.in dem.grd 1" be rerun again?
Thanks

**Comment by Xiaohua-Eric-Xu** [comment]
nope, mode 1 is used to help you decide which image to be the master for alignment purpose. After changing it, you could just run with mode 2.

**Reply by AliRoozban** [reply]
Thank you
